data-structures.md

```diff
@@ -184,7 +184,18 @@ type Signature struct {
 
 Note: As signatures should always be within wrapper types, length prefixing is not needed here.
 
+## FaultSet
+
+FaultSets are used to denote which sectors failed at which block height.
+
+```go
+type FaultSet struct {
+	Index    uint64
+	BitField BitField
+}
+```
 
+The `Index` field is a block height offset from the start of the miners proving period (in order to make it more compact).
```

Comment:
how does this work? unclear how a single index is enough

Reply:
Its an offset from the miners proving period start, which is a number stored on-chain in their miner actor. It corresponds to the set of challenges being sourced from that block height and says that "for this set of proofs, these sectors were not available"